indico/htdocs/js/utils/forms.js

```diff
@@ -36,5 +36,12 @@
             var passwordField = $(this.form).find('input[name="' + confirmField.data('confirmPassword') + '"]');
             validatePasswordConfirmation(passwordField, confirmField);
         });
+
+        $('form [data-disabled-until-change]').closest('form').each(function() {
+            $(this).data('serialized', $(this).serialize());
+        }).on('change input', function() {
+            var changed = $(this).serialize() == $(this).data('serialized');
+            $(this).find('[data-disabled-until-change]').prop('disabled', changed);
+        }).find('[data-disabled-until-change]').prop('disabled', true);
```

Comment:
`.end()` instead of `.find('[data-disabled-until-change]')` will be more efficient here (since you effectively go back to the selector you used before calling `closest`)